README.rst

```diff
@@ -10,6 +10,9 @@ Experimental Data
 
 The experimental data in this repository is currently stored in the `.mdm` file format.
 
+**Other data**
+
+* (2022-Nov-1) New experimental data by EPFL is stored in the `.csv` format at `sky130_fd_pr/epfl <./sky130_fd_pr/epfl>`_. The corresponded notebook can be found at `notebooks/electrical_characterization_sky130_epfl.ipynb <./notebooks/electrical_characterization_sky130_epfl.ipynb>`_.
```

Comment:
maybe move the data under a different top level dirrectory for consistency?
```
sky130_fd_pr_epfl/cells
```
@mithro any input there?

Reply:
Got it

Reply:
Fixed in [be9ac42](https://github.com/google/skywater-pdk-sky130-raw-data/pull/23/commits/be9ac42dd47947dc63d815e460dea4e5a2586848)